internal/asm/f64/ge_test.go

```diff
@@ -196,11 +188,7 @@ func BenchmarkGer(t *testing.B) {
 				x, y, a := gerData(m, n, inc.x, inc.y)
 				b.ResetTimer()
 				for i := 0; i < b.N; i++ {
-					Ger(uintptr(m), uintptr(n),
-						alpha,
-						x, uintptr(inc.x),
-						y, uintptr(inc.y),
-						a, uintptr(n))
+					Ger(uintptr(m), uintptr(n), alpha, x, uintptr(inc.x), y, uintptr(inc.y), a, uintptr(n))
```

Comment:
At the call site is fine to split over lines, but don't both changing this back.